charms/openfga-k8s/src/charm.py

```diff
@@ -169,8 +171,13 @@ def _on_stop(self, _):
         """Stop OpenFGA."""
         container = self.unit.get_container(WORKLOAD_CONTAINER)
         if container.can_connect():
-            container.stop("openfga")
-        self._ready()
+            try:
+                service = container.get_service(SERVICE_NAME)
+            except ModelError:
+                logger.warning("service not found, won't stop")
+                return
+            if service.is_running():
+                container.stop("openfga")
```

Comment:
Shouldn't this be 
` container.stop(SERVICE_NAME)`

Reply:
Ah yeah good catch